<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>equipTab</class>
 <widget class="QWidget" name="equipTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>655</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777210</width>
    <height>16777210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="swView">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="editMode">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="horizontalSpacing">
          <number>-1</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Main</string>
           </property>
           <property name="buddy">
            <cstring>txtMain</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Sub</string>
           </property>
           <property name="buddy">
            <cstring>txtSub</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Range</string>
           </property>
           <property name="buddy">
            <cstring>txtRanged</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Ammo</string>
           </property>
           <property name="buddy">
            <cstring>txtAmmo</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="txtMain"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="txtSub"/>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="txtRanged"/>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="txtAmmo"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Head</string>
           </property>
           <property name="buddy">
            <cstring>txtHead</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Neck</string>
           </property>
           <property name="buddy">
            <cstring>txtNeck</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>L.Ear</string>
           </property>
           <property name="buddy">
            <cstring>txtLEar</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>R.Ear</string>
           </property>
           <property name="buddy">
            <cstring>txtREar</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="txtHead"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="txtNeck"/>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="txtLEar"/>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="txtREar"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_12">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Body</string>
           </property>
           <property name="buddy">
            <cstring>txtBody</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_10">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Hands</string>
           </property>
           <property name="buddy">
            <cstring>txtHands</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>L.Ring</string>
           </property>
           <property name="buddy">
            <cstring>txtLRing</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="label_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>R.Ring</string>
           </property>
           <property name="buddy">
            <cstring>txtRRing</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="txtBody"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="txtHands"/>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="txtLRing"/>
         </item>
         <item row="5" column="3">
          <widget class="QLineEdit" name="txtRRing"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_16">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
           <property name="buddy">
            <cstring>txtBack</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_15">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Waist</string>
           </property>
           <property name="buddy">
            <cstring>txtWaist</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_14">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Legs</string>
           </property>
           <property name="buddy">
            <cstring>txtLegs</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QLabel" name="label_13">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Feet</string>
           </property>
           <property name="buddy">
            <cstring>txtFeet</cstring>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLineEdit" name="txtBack"/>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="txtWaist"/>
         </item>
         <item row="7" column="2">
          <widget class="QLineEdit" name="txtLegs"/>
         </item>
         <item row="7" column="3">
          <widget class="QLineEdit" name="txtFeet"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:15pt; font-weight:600; text-decoration: underline;&quot;&gt;Scripts&lt;/span&gt;&lt;span style=&quot; font-size:15pt;&quot;&gt; -&lt;/span&gt; Add extra commands to your script here.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="txtExtraData">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Command Type:</string>
           </property>
           <property name="buddy">
            <cstring>cmbExtraType</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QStackedWidget" name="swExtraData">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QHBoxLayout" name="_3">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QComboBox" name="cmbExtraCommand">
               <item>
                <property name="text">
                 <string>Job Ability</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Spell or Song</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Self-Notice (echo)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Tell Party</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Tell Linkshell</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbExtraParam">
               <property name="editable">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string>Provoke</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Sneak Attack</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Trick Attack</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbExtraTarget">
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string>&lt;t&gt;</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>&lt;bt&gt;</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>&lt;st&gt;</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>&lt;me&gt;</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QHBoxLayout" name="_4">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QSpinBox" name="spnTimeout">
               <property name="frame">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight</set>
               </property>
               <property name="maximum">
                <number>900</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbTimetype">
               <item>
                <property name="text">
                 <string>Seconds</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Minutes</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Hours</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Command Parameters:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="cmbExtraType">
           <item>
            <property name="text">
             <string>FFXI Command (input)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pause</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QToolButton" name="tbAddToBin">
           <property name="text">
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Command
(or pause time):</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="viewOutput">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="txtTextMode">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtMain</tabstop>
  <tabstop>txtSub</tabstop>
  <tabstop>txtRanged</tabstop>
  <tabstop>txtAmmo</tabstop>
  <tabstop>txtHead</tabstop>
  <tabstop>txtNeck</tabstop>
  <tabstop>txtLEar</tabstop>
  <tabstop>txtREar</tabstop>
  <tabstop>txtBody</tabstop>
  <tabstop>txtHands</tabstop>
  <tabstop>txtLRing</tabstop>
  <tabstop>txtRRing</tabstop>
  <tabstop>txtBack</tabstop>
  <tabstop>txtWaist</tabstop>
  <tabstop>txtLegs</tabstop>
  <tabstop>txtFeet</tabstop>
  <tabstop>txtExtraData</tabstop>
  <tabstop>cmbExtraType</tabstop>
  <tabstop>cmbExtraCommand</tabstop>
  <tabstop>cmbExtraParam</tabstop>
  <tabstop>cmbExtraTarget</tabstop>
  <tabstop>tbAddToBin</tabstop>
  <tabstop>txtTextMode</tabstop>
  <tabstop>cmbTimetype</tabstop>
  <tabstop>spnTimeout</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cmbExtraType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>swExtraData</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
